.circleci/config.yml

```diff
@@ -274,37 +274,6 @@ jobs:
 
 workflows:
   version: 2
-  beta-release:
-    jobs:
-      - dependencies:
-          filters:
-            tags:
-              only:
```

Comment:
is this filter missing in the new config?

Reply:
https://github.com/corona-warn-app/cwa-app-ios/pull/4429/files/8ff2d28fe57d2546c3050e6163b0198b743d5748#diff-150d88cc502b284bc9405bca3a8d7aa5c2f45f6a44dddeb994ec9843bc4236e0R5 nope :D

Reply:
ah okay I did not see this